src/main/java/com/eternalcode/combat/CombatCommand.java

```diff
@@ -25,101 +30,167 @@ public class CombatCommand {
-        UUID secondTargetUniqueId = secondTarget.getUniqueId();
-        UUID playerUniqueId = player.getUniqueId();
-
-        if (playerUniqueId.equals(firstTargetUniqueId) || playerUniqueId.equals(secondTargetUniqueId)) {
-            this.announcer.sendMessage(player, messages.admin.adminCannotTagSelf);
+        
+        if (sender.equals(firstTarget) || sender.equals(secondTarget)) {
+            this.announcer.sendMessage(sender, messages.admin.adminCannotTagSelf);
             return;
         }
-
-        this.fightManager.tag(firstTargetUniqueId, combatTime);
-        this.fightManager.tag(secondTargetUniqueId, combatTime);
+        
+        boolean isTaggedFirst = this.fightManager.tag(firstTarget.getUniqueId(), combatTime, CauseOfTag.COMMAND);
+        boolean isTaggedSecond = this.fightManager.tag(secondTarget.getUniqueId(), combatTime, CauseOfTag.COMMAND);
 
         Formatter formatter = new Formatter()
             .register("{FIRST_PLAYER}", firstTarget.getName())
             .register("{SECOND_PLAYER}", secondTarget.getName());
 
         String format = formatter.format(messages.admin.adminTagMultiplePlayers);
-        this.announcer.sendMessage(player, format);
 
-        this.announcer.sendMessage(firstTarget, messages.playerTagged);
-        this.announcer.sendMessage(secondTarget, messages.playerTagged);
+        if (isTaggedFirst) {
+            this.announcer.sendMessage(firstTarget, messages.playerTagged);
+        }
+        else {
+            this.announcer.sendMessage(sender, messages.admin.cannotTagPlayer);
+        }
+
+        if (isTaggedSecond) {
+            this.announcer.sendMessage(secondTarget, messages.playerTagged);
+        }
+        else {
+            this.announcer.sendMessage(sender, messages.admin.cannotTagPlayer);
+        }
+
+        if (isTaggedFirst || isTaggedSecond) {
+            this.announcer.sendMessage(sender, format);
+        }
+    }
+
+    @Async
+    @Execute(route = "reload")
+    @Permission("eternalcombat.reload")
+    void execute(CommandSender player) {
+        this.configService.reload();
+        this.announcer.sendMessage(player, this.config.messages.admin.reload);
     }
 
     @Execute(route = "untag", required = 1)
     @Permission("eternalcombat.untag")
-    void untag(Player player, @Arg Player target) {
+    void untag(Player sender, @Arg Player target) {
         UUID targetUniqueId = target.getUniqueId();
 
         if (!this.fightManager.isInCombat(targetUniqueId)) {
-            this.announcer.sendMessage(player, this.config.messages.admin.adminPlayerNotInCombat);
+            this.announcer.sendMessage(sender, this.config.messages.admin.adminPlayerNotInCombat);
             return;
         }
 
         this.announcer.sendMessage(target, this.config.messages.playerUntagged);
 
-        this.fightManager.untag(targetUniqueId);
+        if (!this.fightManager.untag(targetUniqueId, CauseOfUnTag.COMMAND)) {
+            return;
```

Comment:
Here it would be useful to send a message to the player that it is not possible - currently, the command will do nothing if it is not possible to untag the target.